src/styles/index.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  padding: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+  color: #060b35;
+  padding-bottom: 4px;
+  padding-left: 16px;
+  padding-right: 16px;
+  max-width: 198px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 140%;
+  color: #616070;
+  padding-bottom: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  line-height: 112%;
+  padding-bottom: 24px;
+  padding-left: 16px;
+  padding-right: 16px;
+  color: #333;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 166px;
+  height: 40px;
+  margin: 0 auto 16px;
```

Comment:
The width of the .card__link could be made flexible instead of fixed to make the layout more responsive.